scripts/ci/validate-schema.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+
```

Comment:
`set -euf -o pipefail` is missing in all scripts 😊  Otherwise, if a command fails, the script will continue to run!